sync/reconcile_pod.go

```diff
@@ -36,12 +40,14 @@ func reconcilePod(clientset *kubernetes.Clientset, watchClient *watch.Client, de
 				if err := deletePod(clientset, watchClient, namespace, podName); err != nil {
 					return nil, err
 				}
+			} else {
+				break
```

Comment:
Somewhat not relevant to this PR, but this piece is needed for `createPod` to not get always get called 5 times.

Reply:
Nice!